lib/teleterm/clusters/cluster_apps.go

```diff
@@ -79,10 +79,7 @@ func (c *Cluster) getApp(ctx context.Context, authClient authclient.ClientI, app
 }
 
 // ReissueAppCert issue new certificates for the app and saves them to disk.
-func (c *Cluster) ReissueAppCert(ctx context.Context, clusterClient *client.ClusterClient, app types.Application) (tls.Certificate, error) {
-	if app.IsAWSConsole() || app.IsGCP() || app.IsAzureCloud() {
-		return tls.Certificate{}, trace.BadParameter("cloud applications are not supported")
-	}
```

Comment:
This was nice to have but not necessary, but it only guarded us from a programmer error – there's no way the user could generate a cert through the UI for these types of apps.